organization/schema.py

```diff
@@ -331,6 +344,70 @@ def mutate(self, info, membership_id, *args, **kwargs):
         return QuitKSGMutation(internal_group_position_membership=membership)
 
 
+class InternalGroupUserHighlightNode(DjangoObjectType):
+    class Meta:
+        model = InternalGroupUserHighlight
+        filter_fields = ["internal_group", "user"]
+        interfaces = (Node,)
+
+    image = graphene.String()
+
+    def resolve_image(self: InternalGroupUserHighlight, info, **kwargs):
+        if self.image:
+            return self.image.url
+        else:
+            return None
+
+    @classmethod
+    def get_node(cls, info, id):
+        return InternalGroupPosition.objects.get(pk=id)
+
+
+class CreateInternalGroupUserHighlightMutation(DjangoCreateMutation):
+    class Meta:
+        model = InternalGroupUserHighlight
+        permissions = ("organization.add_internalgroupuserhighlight",)
+
+
+class PatchInternalGroupUserHighlightMutation(DjangoPatchMutation):
+    class Meta:
+        model = InternalGroupUserHighlight
+        permissions = ("organization.change_internalgroupuserhighlight",)
+
+
+class DeleteInternalGroupUserHighlight(DjangoDeleteMutation):
+    class Meta:
+        model = InternalGroupUserHighlight
+        permissions = ("organization.delete_internalgroupuserhighlight",)
+
+
+class BatchPatchInternalGroupUserHighlightMutation(DjangoBatchPatchMutation):
+    class Meta:
+        model = InternalGroupUserHighlight
+        permissions = ("organization.change_internalgroupuserhighlight",)
+
+
+class InternalGroupUserHighlightQuery(graphene.ObjectType):
+    all_internal_group_user_highlights = graphene.List(InternalGroupUserHighlightNode)
+    internal_group_user_highlight = Node.Field(InternalGroupUserHighlightNode)
+    internal_group_user_highlights_by_internal_group = graphene.List(
+        InternalGroupUserHighlightNode, internal_group_id=graphene.ID()
+    )
+
+    @gql_has_permissions("organization.view_internalgroupuserhighlight")
+    def resolve_all_internal_group_user_highlights(self, info, *args, **kwargs):
+        return InternalGroupUserHighlight.objects.all()
+
+    @gql_has_permissions("organization.view_internalgroupuserhighlight")
```

Comment:
Change to login required

Reply:
<img src="https://media2.giphy.com/media/l0HlVWTRu7Nu6ZU2I/giphy.gif"/>